app.ts

```diff
@@ -1,15 +1,28 @@
 import { app } from 'mu';
 import { mandatarissenRouter } from './routes/mandatarissen';
-import express, { ErrorRequestHandler } from 'express';
+import express, { Request, ErrorRequestHandler } from 'express';
+import bodyParser from 'body-parser';
+
 import { burgemeesterRouter } from './routes/burgemeester-benoeming';
 import { installatievergaderingRouter } from './routes/intallatievergadering';
+import { mandateesDecisionsRouter } from './routes/mandatees-decisions';
+
+app.use(
```

Comment:
This is needed so we can get the information (`req.body`) passed by the delta notifier callback